_site/2016/06/10/ccc-init-we-are-launching-join-us.html

```diff
@@ -30,20 +30,25 @@
 
   <ul>
     <li>
-      <a href="http://localhost:4000/coding-cafe.html">
-        <strong>Coding Cafe</strong>
-      </a>
-    </li>
-    <li>
       <a href="http://localhost:4000/blog.html">
         <strong>Blog</strong>
       </a>
     </li>
     <li>
+        <a href="http://localhost:4000/internships.html">
```

Comment:
Should we not use `bundle exec jekyll build`, in order to get the *published* version? But, since it works, I don't get why we need the _site directory in the first place.